rb/lib/selenium/webdriver/common/manager.rb

```diff
@@ -56,7 +56,7 @@ def add_cookie(opts = {})
         opts[:httpOnly] = http_only if http_only
 
         obj = opts.delete(:expires)
-        opts[:expiry] = seconds_from(obj).to_i if obj
+        opts[:expiry] = seconds_from(obj).to_int if obj
```

Comment:
What's the difference between to_i and to_int? Should we instead use `Integer()`?

Reply:
In this scenario the challenge is that seconds_from can return a Numeric type and the Numeric class only has to_int

RBS signature for seconds_from:

`def seconds_from: (Time | DateTime | Numeric obj) -> (Float | Numeric)`

References for Numeric:

https://ruby-doc.org/core-2.5.1/Numeric.html

Here is a nice article on also different usages:

https://www.rubyfleebie.com/2007/04/05/to_i-vs-to_int/

Error with to_i:

<img width="962" alt="Screenshot 2024-03-13 at 22 21 24" src="https://github.com/SeleniumHQ/selenium/assets/33221555/5177ee57-d895-4a65-9813-9f6bde24787b">

Reply:
Interesting, TIL.